codec-http2/src/main/java/io/netty/handler/codec/http2/internal/hpack/HuffmanDecoder.java

```diff
@@ -213,4 +137,96 @@ private static void insert(Node root, int symbol, int code, byte length) {
             current.children[i] = terminal;
         }
     }
+
+    private static final class DecoderProcessor implements ByteProcessor {
+        private final ByteArrayOutputStream baos = new ByteArrayOutputStream();
```

Comment:
we could also add a `arrayUnsafe()` operation and return an `AsciiString` from `end()` which represents the subsection of `byte[]` that was actually used.